lib/cronofy/client.rb

```diff
@@ -799,6 +840,8 @@ def add_to_calendar(args = {})
 
       body[:event][:start] = encode_event_time(body[:event][:start]) if body[:event][:start]
       body[:event][:end] = encode_event_time(body[:event][:end]) if body[:event][:end]
+      body[:availability] = args[:availability] if args[:availability]
```

Comment:
Ah I've implemented this on a branch already I'll merge my changes with this as you've made a good start on the docs